deployinggpucontainers.mdx

```diff
@@ -0,0 +1,95 @@
+---
+title: 'Deploying GPU Containers'
+---
+### Background
+Accessing affordable GPU compute often gets you out of flow. Big clouds have complicated UI's and high prices, but 3rd party clouds have no supply guarantees.
+Now with the Shadeform API, you can spin up GPU containers on the most affordable compute without leaving your IDE.
+
+### Benchmarking Models
+
+With the Container Launch config, we can spin up a secure VM, and start a container on it automatically. Let's start with a very simple example - running evals on a new model from Huggingface. 
+
+We have an image that does this with the LM Eval Harness library using HF Transformers. The code for it can be found and forked [here](https://github.com/shadeform/examples/tree/main/lm-eval-harness-benchmarking).
+We also have a [notebook](https://github.com/shadeform/examples/blob/main/deploy_container.ipynb) that automatically launches this image on the most affordable A6000's on the market to run the 
+[Mistral-7b-Instruct-v0.2](https://huggingface.co/mistralai/Mistral-7B-Instruct-v0.2) model though a benchmark. You can configure the model, gpu type, number of fewshot examples, and the benchmark eval.
+
+### Setup
+
+The requirements are simple, so in your preferred python environment:
+
+```bash
+git clone https://github.com/shadeform/examples.git
+cd examples/
+```
+Then in `deploy_container.ipynb` input your Shadeform API Key from [here](https://platform.shadeform.ai/settings/api), and a  `huggingface_token` if the model you need is gated (like Llama2 or Gemma).
```

Comment:
We don't support a huggingface api token on the settings api page

Reply:
The customer will have to pass it into their docker container using the environment variable supported by the specific container

Reply:
Okay, will edit for clarity. What I mean is that into the notebook, they will need to put 2 different things: 1. The api key from Shadeform, that they can get on your page, and 2 a hugging face token iff they need to use a llama or gemma model that need authorized access.

Reply:
The container launch cell in the notebook shows this